user_guide/intro_log_in_and_edit_profile.rst

```diff
@@ -0,0 +1,554 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Emails can be sent from user to user within OroCRM and outside OroCRM using internal OroCRM SMTPserver. However, most 
+OroCRM users already have some external mailbox used for work-related purposes. You can synchronize this mailbox with 
+your mailbox in OroCRM in the *"Email synchronization settings"* section.
+
+|
+
+.. image:: ./img/intro/email_sync_1.png
+
+|
+
+To synchronize your existing mailbox with your mailbox in OroCRM, go the the *"Email synchronization settings"* section.
+
+OroCRM can be synchronized with any IMAP/SMTP servers. A dedicated *"Gmail"* synchronization is available to simplify 
+synchronization with Gmail-based accounts. 
+If synchronization with an IMAP server has been defined, all the emails from synchronized folders of the external 
+mailbox will be available to you in *"My Emails"* in OroCRM. If If synchronization with an SMTP server has been defined, 
+all the emails sent from OroCRM will be available in the external mailbox.
+Please note, that if SMTP synchronization has been enabled, your addressee will always see the mailbox address as the 
+*"from"* field value.
+
+.. note::
+
+    If no synchronization has been enabled, emails received from other OroCRM users will appear in the *"Activity"* 
+    section of the *"My User"* page but not in *"My Emails"*. Emails from a synchronized mailbox can be reached from the 
+    both.
+
+Synchronize with any IMAP/SMTP Server
+"""""""""""""""""""""""""""""""""""""    
+
+The functionality can be used to synchronize any IMAP/SMTP server with your mailbox in OroCRM. 
+
+- Select the Account Type - *"Other"* (if available). If this is an only option enabled for the system, the selector 
+  won’t be displayed - skip the step.
+  
+  |
+  
+  |EmailSync2|
+
+  |
+  
+- In order to **receive emails** from the external mailbox in OroCRM:
+
+ - Check the *"Enable IMAP"* box.
+
+ - Provide your IMAP credentials: host, port, and encryption type (contact your administrator for assistance).
+
+- In order to **synchronize emails sent** from OroCRM in to the external mailbox:
+
+ - Check the *"Enable SMTP"* box.
+
+ - Provide your SMTP credentials: host, port, and encryption type (contact your administrator for assistance).
+
+.. hint::
+
+    If you choose not to enable SMTP synchronization, you will still be able to send emails from OroCRM, but they won’t 
+    be synced with the email server and you will not see them in other email clients (such as Outlook or Gmail web 
+    interface). We strongly recommend to enable SMTP at all times.
+
+- In order to **finish the synchronization**:
+
+ - Provide your access credentials: your login/username (usually the email address itself) and password used to access 
+   the mailbox.
+   
+ - Click the :guilabel:`Check Connection/Retrieve Folders` button. 
+
+ - Once connection has been established , you will see the list of folders. Check the folders you want to synchronize 
+   with OroCRM.
```

Comment:
Edit: Once a connection has been established, you will see a list of folders. Check the folders you want to synchronize with OroCRM.